otelcolbuilder/.otelcol-builder.yaml

```diff
@@ -243,5 +242,6 @@ replaces:
   # version which gets then translated into a replace in go.mod file.
   # This does not replace the version that sumologicexporter depends on.
   - github.com/SumoLogic/sumologic-otel-collector/pkg/extension/opampextension => ../../pkg/extension/opampextension
+  - github.com/open-telemetry/opentelemetry-collector-contrib/extension/sumologicextension v0.108.0 => github.com/SumoLogic/opentelemetry-collector-contrib/extension/sumologicextension bf012a09b9651cddda4a907ad4ff17834a0b81ef
```

Comment:
does this mean the hash would need to be changed for every new version release? or is it here just for this PR 🤔

Reply:
We're using a fork of upstream at v0.108.0. Upstream is already on 0.113.0. We'll need to get the changes on our fork into upstream. We can stop using our fork once we update to the next version they release with our changes.